bucket/vroid-studio.json

```
@@ -0,0 +1,32 @@
+{
+    "version": "0.11.3",
+    "description": "A 3D character maker that lets you use your imagination to easily create your own original characters.",
+    "homepage": "https://vroid.com/en/studio",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://policies.pixiv.net/en.html#vroidstudio"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.vroid.com/dist/VRoidStudio-v0.11.3-win.zip",
+            "hash": "ff41ff00954ab7159412d8369b83dfbba617f96fb2b84a783b9254e1a4213cae",
+            "bin": "VRoidStudio.exe",
+            "shortcuts": [
+                [
+                    "VRoidStudio.exe",
+                    "VRoid Studio"
+                ]
+            ]
+        }
+    },
+    "extract_dir": "VRoidStudio-v0.11.3-win",
+    "checkver": "dist/VRoidStudio-v([\\d.]+)-win.zip",
```

Adding shim does not make sense. There is no command line support

```suggestion
    "shortcuts": [
        [
            "VRoidStudio.exe",
            "VRoid Studio"
        ]
    ],
    "checkver": "dist/VRoidStudio-v([\\d.]+)-win\\.zip",
```

---

True, I removed the "bin" attribute.